analyzer/src/funTest/assets/projects/external/example-python-flask-expected-output.yml

```diff
@@ -33,26 +33,23 @@ packages:
 - id: "PyPI::click:8.1.3"
   purl: "pkg:pypi/click@8.1.3"
   authors:
-  - "Armin Ronacher"
+  - "Armin Ronacher <armin.ronacher@active-4.com>"
   declared_licenses:
-  - "BSD License"
```

Comment:
python-inspector is not using the classifiers to gather license information, see for example:

https://pypi.org/pypi/click/6.7/json
This does only have "BSD License" in the classifier but no license property.

https://pypi.org/pypi/rdflib/5.0.0/json
This does have a declared license "BSD-3-Clause" which I think we ignored in the old implementation, because we only used information from PyPI to fill empty fields.

I will create a feature request with python-inspector to provide the classifiers once I'm done with upgrading.

Reply:
I've added the "breaking change" label, as classifiers are not used, so declared licenses may differ which in turn may break declared license mappings (ort configuration).

Reply:
It would be nice to have the issue about parsing classifiers filed before the merge of this PR, so the issue could be referenced from the "Pip: Capture metadata using python-inspector" commit to explain the diff in the expected results.

Reply:
Done.